### PR TITLE
Created navigation test

### PR DESCRIPTION
**Pull request by yaichenbaum**
Created a test to navigate to a directory

### CODE REVIEW

File: tests/Files.InteractionTests/Tests/NavigationTests.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		[TestCleanup]
+		public void Cleanup()
+		{
+			TestHelper.InvokeButtonByName("Home");
+		}
+
+		[TestMethod]
+		public void VerifyNavigationWorks()
+		{
+			TestHelper.InvokeButtonByName("Desktop");
```
Comment (chingucoding) [inline tests/Files.InteractionTests/Tests/NavigationTests.cs:20]: Was there a specific reason why you chose to invoke by name here?

Comment (yaichenbaum) [reply tests/Files.InteractionTests/Tests/NavigationTests.cs:20]: It's the first thing I tried and it worked.

Comment (chingucoding) [reply tests/Files.InteractionTests/Tests/NavigationTests.cs]: Probably should have said this, but you would also need to set the AutomationProperties.AutomationID property on the XAML controls.

Comment (chingucoding) [reply tests/Files.InteractionTests/Tests/NavigationTests.cs]: Now you have updated the AutomationID to the same value as the text again which again will fail with localization. What I have done for the settings test is set the AutomationID to a fixed value regardless of locale. Sorry about the confusion.

Comment (yaichenbaum) [reply tests/Files.InteractionTests/Tests/NavigationTests.cs]: The items aren't fixed as they're from an item source, how would I set the fixed value?

Comment (chingucoding) [reply tests/Files.InteractionTests/Tests/NavigationTests.cs]: Oh shoot, totally missed that. That's a very difficult thing. There are a few hacky workarounds we could use but I think the best would be to either accept this issue with localization or set the automation ID of the parent control and grab the first child (assuming we don't care which folder to open here).

On a side note, it seems that it should be `AutomationProperties.AutomationId` with a lowercase d, sorry forgot that.

Comment (yaichenbaum) [reply tests/Files.InteractionTests/Tests/NavigationTests.cs]: One way to work around the localization issue would be to create and pin a custom folder as part of the test.

Comment (yaichenbaum) [reply tests/Files.InteractionTests/Tests/NavigationTests.cs]: > or set the automation ID of the parent control and grab the first child (assuming we don't care which folder to open here).

This sounds like a better idea, is there a way to do that?

Comment (chingucoding) [reply tests/Files.InteractionTests/Tests/NavigationTests.cs]: Yes that is possible, you can query the sub children of an element by calling `FindElementsBy...` on the selenium object (in our case the element with the AutomationID).

**Review by chingucoding — APPROVED** [review]

**Pull request merged**